divvy/cloudfilebrowser.js

```diff
@@ -860,55 +854,14 @@ var cloudFileBrowser = (function() {
                 }, callbackArgs);
             });
 
-            $('.listTable ul li.filename').on('click', function (event) {
+            $('.listTable ul li.filename').on('dblclick', function (event) {
                 event.preventDefault();
                 event.stopPropagation();
 
-                var fileId = $(this).closest('ul').data('file-id');
-                var fileInfo = '#file-info';
-                var fileName = $(this).text();
                 var location = $(this).next().text();
-                var listHTML = '<ul><li>Filename:</li><li>' + fileName + '</li></ul>' +
-                                '<ul><li>Location:</li><li>' + location + '</li></ul>' +
-                                '<a href="#" class="selectbutton" data-file-id="' +
-                                fileId +'">Select File</a>';
-
-                extension = fileName.split('.').pop();
-
-                //Get the thumbnail of the image only when the extension is of type image
-                $('#file-info .preview img').remove();
-
-                var extlower = extension.toLowerCase();
-                if (extlower == "jpg" | extlower == "gif" | extlower == "jpeg" | extlower == "png")
-                {
-                    // Prepare thumbnail to be displayed
-                    provision.displayFile(element, location, cloudFileBrowser.displayThumbnail);
-                }
-
-                $(fileInfo).addClass('show').find('.fileDetails').html(listHTML);
-
-
-                $(fileInfo).find('.selectbutton').on('click', function (event) {
-                    event.preventDefault();
-                    event.stopPropagation();
-
-                    provision.fileSelected(element, location, fileId);
-                });
-
-                $(fileInfo).find('.downloadbutton').on('click', function (event) {
-                    event.preventDefault();
-                    event.stopPropagation();
-
-                    provision.downloadFile(element, location);
-                });
-
-            });
-
-            $('div.preview a.close').on('click', function (event) {
-                event.preventDefault();
-                event.stopPropagation();
+                var fileId = $(this).closest('ul').data('file-id');
 
-                $('#file-info').removeClass('show');
+                provision.fileSelected(element, location, fileId);
```

Comment:
What's the difference between `fileSelected` and `downloadFile`?

Reply:
They're different

Reply:
I believe this is still outstanding...